portfolio/src/main/webapp/passions.html

```diff
@@ -81,5 +79,45 @@ <h4 class="pass-header">Chess</h4>
         </div>
       </div>
     </section>
+    <!-- Slideshow Section -->
+    <section class="page-section" id="sldeshow">
+      <div class="container">
+        <div class="text-center">
+          <h2 class="section-heading text-uppercase">Travel gallery</h2>
+          <h3 class="section-subheading text-muted">Here is a brief list with some of the things I enjoy doing in my spare time</h3>
+        </div>
+        <div class="slideshow-container">
+          <div class="slides my-fade">
+            <div class="numbertext">1 / 3</div>
+            <img src="images/delta-danube.jpg" class="rounded">
+            <div class="text">Delta Danube</div>
+          </div>
+
+          <div class="slides my-fade">
+            <div class="numbertext">2 / 3</div>
+            <img src="images/fagaras.jpg" class="rounded">
+            <div class="text">Fagaras Moutains</div>
+          </div>
+
+          <div class="slides my-fade">
+            <div class="numbertext">3 / 3</div>
+            <img src="images/sinaia.jpg" class="rounded">
+            <div class="text">Sinaia</div>
+          </div>
+          <a class="prev" onclick="plusSlides(-1)">&#10094;</a>
+          <a class="next" onclick="plusSlides(1)">&#10095;</a>
+        </div>
+
+        <br>
+
+        <div style="text-align:center">
```

Comment:
I'd avoid inline styles, see https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns.

Reply:
It only had debugging purposes, but I forgot to delete it after.

Reply:
It only had debugging purposes, but I forgot to delete it after.